cluster/defaults.go

```diff
@@ -902,13 +905,17 @@ func (c *Cluster) setClusterNetworkDefaults() {
 		setDefaultIfEmpty(&c.Network.AciNetworkProvider.OpflexAgentOvsAsyncjsonEnabled, DefaultAciOpflexAgentOvsAsyncjsonEnabled)
 		setDefaultIfEmpty(&c.Network.AciNetworkProvider.OpflexAgentPolicyRetryDelayTimer, DefaultAciOpflexAgentPolicyRetryDelayTimer)
 		setDefaultIfEmpty(&c.Network.AciNetworkProvider.AciMultipod, DefaultAciAciMultipod)
-		setDefaultIfEmpty(&c.Network.AciNetworkProvider.OpflexDeviceReconnectWaitTimeout, DefaultAciOpflexDeviceReconnectWaitTimeout)
```

Comment:
This removed field is still referenced by `pkg/rke/templates/aci-v6.0.3.2.go` template introduced in https://github.com/rancher/kontainer-driver-metadata/pull/1291. Do we have to remove existing fields, especially ones still referenced by the template?

@kinarashah , tagging you as you have more context on what could happen if we remove fields from types.

Reply:
The variable is not removed, just the setting default parts. So other places it is still there.